runtime/queries/groovy/highlights.scm

```diff
@@ -0,0 +1,96 @@
+(unit
+  (identifier) @variable)
+
+(string
+  (identifier) @variable)
+
+(escape_sequence) @constant.character.escape
+
+(block
+  (unit
+    (identifier) @namespace))
+
+(func
+  (identifier) @function)
+
+(number) @constant.numeric
+
+((identifier) @constant.builtin.boolean
+  (#any-of? @constant.builtin.boolean "true" "false"))
+
+((identifier) @constant
+  (#match? @constant "^[A-Z][A-Z\d_]*$"))
```

Comment:
```suggestion
  (#match? @constant "^[A-Z][A-Z\\d_]*$"))
```

I believe we need to escape backslashes because of the way strings are intpreted in the query languag